src/main/java/org/elasticsearch/cluster/metadata/MetaData.java

```diff
@@ -642,19 +642,11 @@ public String resolveIndexRouting(@Nullable String routing, String aliasOrIndex)
      * @param indicesOptions   how the aliases or indices need to be resolved to concrete indices
      * @param aliasesOrIndices the aliases or indices to be resolved to concrete indices
      * @return the obtained concrete indices
-<<<<<<< HEAD
```

Comment:
yeah, that is actually from master. pushed a fix:  24ffcc28ac67dbd9ca6e27642121af17f7aaced7